### PR TITLE
New 8 Tips added

### PR DESCRIPTION
**Pull request by Glebsky**
Added new tips in DB Models and Eloquent (55 tips), Models Relations (30 tips) categories.

### CODE REVIEW

**Comment by Glebsky** [comment]
By mistake, I deleted the link to Twitter, one of the participants. Fixed this error. in Fix commit

**Comment by PovilasKorop** [comment]
@Glebsky wow thank you!

File: README.md [hunk 1]
```diff
@@ -869,9 +876,134 @@ DB::unprepared(
...
+    return Post::findOrFail($id)->tags()->orderPivotBy('flag', 'desc')->get();
+}
+```
+
+Tip given by [@PascalBaljet](https://twitter.com/pascalbaljet)
+
+### Find a single record from a database
+The `sole()` method will return only one record that matches the criteria. If no such entry is found, then a `NoRecordsFoundException` will be thrown. If multiple records are found, then a `MultipleRecordsFoundException` will be thrown.
+```php
+DB::table('products')->where('ref', '#123')->sole()
```
Comment (kang8) [inline README.md:935]: There is no line end semicolon.

**Review by kang8 — COMMENTED** [review]
Some suggest.

**Comment by PovilasKorop** [comment]
@kang8 thank you, fixed. Next time, please submit a Pull Request instead of a comment, if you can.

**Comment by kang8** [comment]
Of course, sure.

File: README.md [hunk 2]
```diff
@@ -870,7 +877,131 @@ DB::unprepared(
...
+    public $table = 'tags';
+}
+
+class Post extends Model
+{
+    public $table = 'posts';
+
+    public function tags()
+    {
+        return $this->belongsToMany(Tag::class, 'posts_tags', 'post_id', 'tag_id')
```
Comment (iammikek) [inline README.md:911]: naming convention for join tables is singular models in alphabetical order. So this

>          return $this->belongsToMany(Tag::class, 'posts_tags', 'post_id', 'tag_id')

should be using table `post_tag` and that would mean you wouldn't need to specify the table name or keys.

>      return $this->belongsToMany(Tag::class)

Comment (PovilasKorop) [reply README.md:911]: @iammikek I would accept a pull request for this change

**Pull request merged**